templates/14.0/odoo.cfg.tmpl

```diff
@@ -0,0 +1,50 @@
+[options]
+addons_path = ${ADDONS_PATH}
+data_dir = /data/odoo
+auto_reload = False
+db_host = ${DB_HOST}
+db_name = ${DB_NAME}
+db_user = ${DB_USER}
+db_password = ${DB_PASSWORD}
+db_port = ${DB_PORT}
+dbfilter = ^${DB_NAME}$
+list_db = False
+admin_passwd = ""
+db_maxconn = ${DB_MAXCONN}
+limit_memory_soft = ${LIMIT_MEMORY_SOFT}
+limit_memory_hard = ${LIMIT_MEMORY_HARD}
+limit_request = ${LIMIT_REQUEST}
+limit_time_cpu = ${LIMIT_TIME_CPU}
+limit_time_real = ${LIMIT_TIME_REAL}
+log_handler = ${LOG_HANDLER}
+log_level = ${LOG_LEVEL}
+max_cron_threads = ${MAX_CRON_THREADS}
+workers = ${WORKERS}
+logfile = ${LOGFILE}
+log_db = ${LOG_DB}
+logrotate = True
+syslog = ${SYSLOG}
+running_env = ${RUNNING_ENV}
+without_demo = ${WITHOUT_DEMO}
+server_wide_modules = ${SERVER_WIDE_MODULES}
+proxy_mode = True
+unaccent = True
+sentry_enabled = ${SENTRY}
+sentry_logging_level = warn
+sentry_exclude_loggers = werkzeug
+sentry_ignore_exceptions = odoo.exceptions.AccessDenied,odoo.exceptions.AccessError,odoo.exceptions.MissingError,odoo.exceptions.RedirectWarning,odoo.exceptions.UserError,odoo.exceptions.ValidationError,odoo.exceptions.Warning,odoo.exceptions.except_orm
+sentry_processors = odoo.addons.sentry.logutils.SanitizeOdooCookiesProcessor
+sentry_transport = threaded
+sentry_include_context = true
+sentry_environment = ${RUNNING_ENV}
+sentry_auto_log_stacks = false
+sentry_odoo_dir = /odoo/src/odoo/
+sentry_dsn = ${SENTRY_DSN}
+sentry_release = ${SENTRY_RELEASE}
+sentry_traces_sample_rate = ${SENTRY_TRACES_SAMPLE_RATE}
+encryption_key_dev=${ENCRYPTION_KEY_DEV}
+encryption_key_ci=${ENCRYPTION_KEY_CI}
+encryption_key_prod=${ENCRYPTION_KEY_PROD}
```

Comment:
Let's think about new environment variables in another PR.

Reply:
yes sure, it's was an example from another project.